README.md

```diff
@@ -28,16 +28,13 @@ Add this dependency to your application pom.xml
 ```
 
```

Comment:
Is the 'configuration' mentioned in the readme?  Might want a brief explanation of where to get the vaultName / tenant id / client id / secret, etc.

Reply:
added